eclair-core/src/test/scala/fr/acinq/eclair/channel/states/ChannelStateTestsHelperMethods.scala

```diff
@@ -119,8 +125,10 @@ trait ChannelStateTestsHelperMethods extends TestKitBase {
     val alice2relayer = TestProbe()
     val bob2relayer = TestProbe()
     val channelUpdateListener = TestProbe()
-    system.eventStream.subscribe(channelUpdateListener.ref, classOf[LocalChannelUpdate])
-    system.eventStream.subscribe(channelUpdateListener.ref, classOf[LocalChannelDown])
+    systemA.eventStream.subscribe(channelUpdateListener.ref, classOf[LocalChannelUpdate])
+    systemA.eventStream.subscribe(channelUpdateListener.ref, classOf[LocalChannelDown])
+    systemB.eventStream.subscribe(channelUpdateListener.ref, classOf[LocalChannelUpdate])
+    systemB.eventStream.subscribe(channelUpdateListener.ref, classOf[LocalChannelDown])
```

Comment:
Isn't it worth splitting the `channelUpdateListener` as well (one for each peer)?

Reply:
I went with the minimal change. I think the whole handing of events in the state tests needs to be reworked.

Reply:
Fair enough, that can be a separate PR later.